tutor/src/hypothesis/index.coffee

```diff
@@ -0,0 +1,23 @@
+
+User = require '../models/user'
+Courses = require '../models/course'
+
+loadSidebar = () ->
+
+startHypothesis = (authority, embedUrl, clientId) ->
+  console.log('Starting Hypothesis ...')
+
+
+start = (hypothesisData) ->
+    console.log(hypothesisData)
+    apiUrl = hypothesisData['api_url']
+    authority = hypothesisData['authority']
+    clientId = hypothesisData['client_id']
+    embedUrl = hypothesisData['embed_url']
+    clientUrl = hypothesisData['client_url']
+    sidebarAppUrl = hypothesisData['sidebar_app_url']
+
+    if !!find(Courses.active, { appearance_code: 'college_biology'})
+        startHypothesis(authority, embedUrl, clientId)
+
+module.exports = {startHypothesis, start}
```

Comment:
FYI, we've been moving from  coffee script to es6 js with `import / export` modules.  Fine to still use coffee though if that's what you're more comfortable with.

Reply:
Also I think this could be written something like:
```
import {mapValues, camelCase } from 'lodash';

let HDATA;

start  = (hypothesisData) ->
  HDATA = mapValues(hypothesisData, camelCase)
  if !!find(Courses.active, { appearance_code: 'college_biology'})
    startHypothesis(authority, HDATA.embedUrl, HDATA.clientId)
```

Reply:
I've been learning coffeescript so i don't mind not using that.

Reply:
Yea, the usage of map seems to be heavily encouraged in js as it is not very widely used in python. I'll make the adjustments.